tools/config-generator/main.go

```diff
@@ -53,9 +51,6 @@ const (
 
 	// generalProwConfig contains config-wide definitions.
 	generalProwConfig = "prow_config.yaml"
```

Comment:
Can we also delete this line?

Reply:
Interesting, this file is gone on my branch